CMakeLists.txt

```diff
@@ -668,20 +668,27 @@ endif()
 
 if(AVIF_BUILD_APPS)
     add_executable(avifenc apps/avifenc.c)
+    if(WIN32)
```

Comment:
Could be factorized in a macro. Then again. `tests/` could not access it. Will refactor later.

Reply:
Thank you. After we know this code works, we can consider using it with all executable programs, including the test programs.